File: solutions/_2390_removing_stars_from_a_string/solution.py
class Solution:
    def removeStars(self, s: str) -> str:
        res = []
        for c in s:
            if res and c == "*":
                res.pop()
            else:
                res.append(c)
        return "".join(res)


# https://leetcode.com/problems/removing-stars-from-a-string/

File: solutions/_2390_removing_stars_from_a_string/solution.js
/**
 * @param {string} s
 * @return {string}
 */
var removeStars = function (s) {
  const res = [];
  for (char of s) {
    if (char === '*') res.pop();
    else res.push(char);
  }
  return res.join('');
};

module.exports = { removeStars };

/*
https://leetcode.com/problems/removing-stars-from-a-string/
*/
